app.css

```diff
@@ -1,3 +1,88 @@
-body {
-  background: purple;
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+
+
+.blog__post {
+  width: 44%;
+  float: left;
+  margin: 2rem;
+  box-shadow:3px 5px 18px rgba(0,0,0,0.5);
+
+}
+
+.blog__head {
+  background-color: #dbbc35;
+  position: relative;
+}
+
+img {
+  border-radius: 50%;
+  position: absolute;
+  transform: translate(-50%);
+  width: 40px;
+  top: 8px;
```

Comment:
Don't try guessing positioning values. Instead always look at using `0`, `+/-50%`, `+/-100%` with `tranform: translate` using similar values to perfectly center items.